File: ProgrammingFundamentals/C#/03.DataTypesAndVariables/Exercises/P17_PrintPartOfASCIITable/P17_PrintPartOfASCIITable.cs
using System;

namespace P17_PrintPartOfASCIITable
{
    class P17_PrintPartOfASCIITable
    {
        static void Main(string[] args)
        {
            int startIndex = int.Parse(Console.ReadLine());
            int endIndex = int.Parse(Console.ReadLine());

            for(int i = startIndex; i <= endIndex; i++)
            {
                Console.Write("" + (char)i + " ");
            }
        }
    }
}

File: ProgrammingFundamentals/Java/03.DataTypesAndVariables/Exercises/src/P17_PrintPartOfASCIITable.java
import java.util.Scanner;
public class P17_PrintPartOfASCIITable {
    public static void main(String[] args) {
        Scanner console = new Scanner(System.in);
        int startIndex = Integer.parseInt(console.nextLine());
        int endIndex = Integer.parseInt(console.nextLine());

        for(int i = startIndex; i <= endIndex; i++)
        {
            System.out.print("" + (char)i + " ");
        }
    }
}
